index.html

```diff
@@ -1,27 +1,48 @@
 <html>
-   <head>
-     <meta charset="utf-8">
-     <meta name="viewport" content="width=device-width, initial-scale=1">
-     <title>learne</title>
-     <link rel="stylesheet" href="//cdn.jsdelivr.net/jquery.slick/1.6.0/slick.css"/>
-     <link rel="stylesheet" href="app.css">
-     <script src="https://use.fontawesome.com/ac8a89b97e.js"></script>
-   </head>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">ש
+    <title>learne</title>
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/jquery.slick/1.6.0/slick.css"/>
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/jquery.slick/1.6.0/slick-theme.css"/>
+    <link rel="stylesheet" href="app.css">
+    <script src="https://use.fontawesome.com/ac8a89b97e.js"></script>
 
-   <body>
+  </head>
 
-     <div class="head-wrapper">
-       <header class="container">
-       <h1><i class="fa fa-graduation-cap" aria-hidden="true"></i> Learne</h1>
-          <ul>
-           <li>שיטת הלימוד</li>
-           <li>לקוחות מספרים</li>
-           <li>צור קשר</li>
-         </ul>
-       </header>
-     </div>
+  <body>
 
-   </body>
-   </body>
- </html>
+    <div class="head-wrapper">
+      <header class="container">
+        <h1><i class="fa fa-graduation-cap" aria-hidden="true"></i> Learne</h1>
+        <ul>
+          <li>שיטת הלימוד</li>
+          <li>לקוחות מספרים</li>
+          <li>צור קשר</li>
+        </ul>
+      </header>
+    </div>
+    <div class="single-item">
+      <div>Pic 1</div>
+      <div>Pic 2</div>
+      <div>Pic 3</div>
+    </div>
+
+    <div class="iconMenuBar">
+      <ul class="container">
+        <li><i class="fa fa-github" aria-hidden="true"></i><br/>
+ללמוד מכל <br/>מקום</li>
```

Comment:
There are two br tags, one between the icon and the text and the other inside the text itself, you want me to delete them both, or only the first one?